JustFixNYC/tenants2
# Internationalize more messages on NoRent.org.

This attempts to internationalize all the remaining messages on NoRent.org that we haven't yet covered, **with the exception of the NoRent letter, emails, and text messages**.

It also fixes #1477, a bug whereby server-rendered static content (modals and PDFs/emails/etc) aren't localized.

## How to manually test this PR

At the terminal, run:

```
node localebuilder.js --garble
yarn lingui:compile
python manage.py compilemessages
```

Now visit NoRent's `/es/` locale and make sure all text that appears is either full of question marks, like the screenshot in #1468, or that it's text that doesn't need localizing (like the "developer" navbar menu).

## To do

- [x] Re-extract message catalogs.
- [x] Ensure compatibility mode UI is internationalized (or punt to another issue).
- [x] Internationalize `validate_phone_number` errors.
